lib/src/vector_drawable.dart

```diff
@@ -1143,6 +1150,9 @@ class DrawableShape implements DrawableStyleable {
       }
     };
 
+    if (style.mask != null) {
+      canvas.saveLayer(bounds, Paint());
```

Comment:
Don't pass in the bounds here.  SaveLayer will just use the current clip anyway, and the bounds may not actually be what we want if there are transforms involved.

Reply:
Also, do we really need both of these?  Is this one required?

Reply:
You mean both invocations of saveLayer? The first one is needed to let te mask only apply on the current shape rendered in the "innerDraw" method, and the second one is needed to convert the mask to transparency and apply it to the layer